static/css/style.css

```diff
@@ -184,41 +211,97 @@ a:hover {
     cursor: text;
 }
 
-.rating {
-    color: #ffaa1d;
+/* Image and image selector */
+
+.overlay-container {
+    position: relative;
+    width: 100%;
+}
+
+.card-img {
+    display: block;
+    width: 100%;
+    height: auto;
+}
+
+.image-overlay {
```

Comment:
Minor nitpick: you abbreviate “image” to “img” in the former selector, but not in this one.